doc/Learn/Http/HttpOverview.md

```diff
@@ -78,7 +78,7 @@ For more information about configuring `HttpClient` with custom endpoint options
 Similarly, **Refit endpoints** can be registered as services and configured in a similar way.
 
 ```csharp
-private IHost Host { get; }
+private IHost Host { get; set; }
```

Comment:
Why is this needed?

Reply:
@nickrandolph on some documentation pages we are setting a value for `Host`
![image](https://github.com/unoplatform/uno.extensions/assets/16295702/52c6b68f-b015-425a-9fb8-990ae5af7356)
Should make sense to have a setter for `Host` or just on the pages where we are setting a value for `Host`

Reply:
```suggestion
private IHost Host { get; private set; }
```

Reply:
Fine by me then

Reply:
Would prefer the `private set`.